python/packages/autogen-ext/tests/models/test_openai_model_client.py

```diff
@@ -2058,4 +2058,31 @@ async def test_add_name_prefixes(monkeypatch: pytest.MonkeyPatch) -> None:
     assert str(converted_mm["content"][0]["text"]) == "Adam said:\n" + str(oai_mm["content"][0]["text"])
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "model",
+    [
+        "gpt-4o-mini",
+        "gemini-1.5-flash",
+        # TODO: Add anthropic models when available.
+    ],
+)
+async def test_muliple_system_message(model: str, openai_client: OpenAIChatCompletionClient) -> None:
```

Comment:
Instead using actual end points we can use mocks see other examples in the same file.

Reply:
Thanks a lot,
however I refactor my code, and do not need mock more.
Nonetheless, I'd like to leave this test code behind so that I can also test its behavior.